R/mod_marg.R

```diff
@@ -35,9 +36,20 @@
 #' mod <- glm(outcome ~ distance + factor(sex),
 #'            data = margex, family = 'binomial')
 #' mod_marg2(mod, var_interest = 'sex', type = 'levels', at = NULL)
+#'
+#' data(margex)
```

Comment:
You should add a note here describing what this example does (i.e., custom variance-covariance matrix).

I would also add to details.